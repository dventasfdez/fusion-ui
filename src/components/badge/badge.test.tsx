import React from 'react';
import renderer from 'react-test-renderer';
import Badge from './badge';

test('Empty badge should render and match snapshot', () => {
  const component = renderer.create(<Badge />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('Badge should render and match snapshot', () => {
  const component = renderer.create(<Badge>8</Badge>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('Badge small should render and match snapshot', () => {
  const component = renderer.create(<Badge small>8</Badge>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('Badge error should render and match snapshot', () => {
  const component = renderer.create(<Badge error>8</Badge>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('Badge error small should render and match snapshot', () => {
  const component = renderer.create(
    <Badge error small>
      8
    </Badge>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('Badge success should render and match snapshot', () => {
  const component = renderer.create(<Badge success>8</Badge>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
test('Badge small should render and match snapshot', () => {
  const component = renderer.create(
    <Badge success small>
      8
    </Badge>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
