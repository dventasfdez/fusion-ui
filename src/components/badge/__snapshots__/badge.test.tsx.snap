// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge error should render and match snapshot 1`] = `
<span
  className="badge_error "
>
  8
</span>
`;

exports[`Badge error small should render and match snapshot 1`] = `
<span
  className="badge_small_error "
>
  8
</span>
`;

exports[`Badge should render and match snapshot 1`] = `
<span
  className="badge "
>
  8
</span>
`;

exports[`Badge small should render and match snapshot 1`] = `
<span
  className="badge_small "
>
  8
</span>
`;

exports[`Badge small should render and match snapshot 2`] = `
<span
  className="badge_small_success "
>
  8
</span>
`;

exports[`Badge success should render and match snapshot 1`] = `
<span
  className="badge_success "
>
  8
</span>
`;

exports[`Empty badge should render and match snapshot 1`] = `
<span
  className="badge_empty "
/>
`;
